packages/charts/src/components/ComposedChart/index.tsx

```diff
@@ -252,12 +258,14 @@ const ComposedChart: FC<ComposedChartProps> = forwardRef((props: ComposedChartPr
               axisProps.yAxisId = index;
               axisProps.width = yAxisWidth;
               AxisComponent = YAxis;
+              axisProps.orientation = useIsRTL() ? 'right' : 'left';
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.